### PR TITLE
Update Storage-component.md

### PR DESCRIPTION
**Pull request by cfjedimaster**
Provides details about my recent PR to allow for unique file names.

### CODE REVIEW

File: pages/en/lb3/Storage-component.md [hunk 1]
```diff
@@ -269,6 +269,10 @@ as shown in the following table.
   </tbody>
 </table>
 
+## Automatic Unique Filenames
+
+As documented above, a file uploaded with the same name as an existing file will be overwritten. By adding the configuration key `nameConflict' with the value `makeUnique`, files will automatically be renamed with a UUID and the existing file extension of the original file name. Most likely this is an option that you will want to enable, otherwise you will need to ensure uniqueness on the code calling the API.
```
Comment (crandmck) [inline pages/en/lb3/Storage-component.md:274]: "nameConflict" needs a closing back-tick not single-quote: s/b `nameConflict`.

**Review by crandmck — CHANGES_REQUESTED** [review]
One small formatting comment, otherwise LGTM.

**Comment by cfjedimaster** [comment]
Ah thanks!

**Comment by crandmck** [comment]
I'll go ahead and land this, and then make the suggested fix.

**Pull request merged**